### DISCUSSION TITLE
How to remove email address in pull requests?

### DISCUSSION THREAD

**Discussion by Newcool1230** [community/community#22093] [New to GitHub]
<p>I was recently added as a co-author in a pull request and it showed my full email address even in a private tab.</p>
<p>Is there a way for me to remove this feature or is there a setting which can hide my actual email address from showing in pull requests?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/newcool1230">@Newcool1230</a> welcome to the community, thank you for joining us.</p>
<p>You may choose to keep your email address as private by <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address">enabling email address privacy</a>, you can choose which verified email address to author changes with when you edit, delete, or create files or merge a pull request on GitHub. If you enabled email address privacy, then the commit author email address cannot be changed and is  <code>&lt;username&gt;@users.noreply.github.com</code>  by default.</p>
<p>To keep your email address private when performing web-based Git operations, click  <strong>Keep my email address private</strong> .</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d51a9000f9e3dbec2e59b50cb7840cc38fd19400.png" data-download-href="/uploads/short-url/upcBTbI55fw1md3OoQgoSO56j5u.png?dl=1" title="Image of checkbox to keep your email address private"><img src="https://user-images.githubusercontent.com/20666190/181068850-0ba9d846-e7c8-4991-a1c8-b87b3639de96.png" alt="Image of checkbox to keep your email address private" data-base62-sha1="upcBTbI55fw1md3OoQgoSO56j5u" width="690" height="194" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d51a9000f9e3dbec2e59b50cb7840cc38fd19400_2_690x194.png, https://aws1.discourse-cdn.com/github/original/2X/d/d51a9000f9e3dbec2e59b50cb7840cc38fd19400.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d51a9000f9e3dbec2e59b50cb7840cc38fd19400.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d51a9000f9e3dbec2e59b50cb7840cc38fd19400_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Image of checkbox to keep your email address private</span><span class="informations">912×257 33.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I hope this helps!</p>

**Comment by Newcool1230** [comment]
<p>Thank you for the help.</p>
<p>What if someone adds me as a co-author before the setting my email as private?</p>
<p>How can I have my email address removed from that?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the follow up, you can contact support directly to make a request <a href="https://support.github.com/contact/privacy">https://support.github.com/contact/privacy</a></p>
